.github/workflows/unset-environment-new-cli.yml

```
@@ -55,34 +55,40 @@ jobs:
         TEST_MODE: true
     - shell: bash
       run: |
-        CPP_DB=${{ fromJson(steps.analysis.outputs.db-locations).cpp }}
+        CPP_DB="${{ fromJson(steps.analysis.outputs.db-locations).cpp }}"
         if [[ ! -d $CPP_DB ]] || [[ ! $CPP_DB == ${{ runner.temp }}/customDbLocation/* ]]; then
-          echo "Did not create a database for CPP, or created it in the wrong location."
+          echo "Did not create a database for CPP, or created it in the wrong location." \
+            "Expected location was '${ runner.temp }'/customDbLocation/* but actual was '${ CPP_DB }'"
```

Careful...`${ CPP_DB }` is not valid. Also, I think `runner.temp` is not correct. You want the actions expression, not a bash variable.

```suggestion
            "Expected location was '${{ runner.temp }}/customDbLocation/*' but actual was '${CPP_DB}'."
```

---

There is also a corresponding env var, which is better since you're already in Bash: `${RUNNER_TEMP}`.

---

Ah, I see, I had gotten the Actions expression syntax mixed up with bash's. I have made those changes!